sale_untaxed_amount_to_invoice_backport/README.rst

```diff
@@ -0,0 +1,3 @@
+=======================================
```

Comment:
This file is generated automatically.
Leave just a single line here: „Will be generated automatically“